exercise_3.py

```diff
@@ -0,0 +1,15 @@
+import numpy as np
+
+
+def get_num_of_columns_and_rows(matrix: np.array) -> tuple:
```

Comment:
Use the most specific return types.
https://docs.python.org/3/library/typing.html